scripts/update_grafana.py

```diff
@@ -0,0 +1,51 @@
+import json
+import logging
+import subprocess
+
+import requests
+
+from azul import config
+logger = logging.getLogger(__name__)
+
+
+def main():
+    logging.basicConfig(format='%(asctime)s %(levelname)-7s %(threadName)-7s: %(message)s', level=logging.INFO)
+    base_url = _get_base_url()
+    dashboard = get_dashboard_json_from_terraform()
+    response = update_dashboard(base_url, dashboard)
+    logging.info(response.json())
+
+
+def _get_base_url():
+    logging.info('Format base URL')
+    user = config.grafana_user
+    password = config.grafana_password
+    domain = config.grafana_endpoint
+    return f"http://{user}:{password}@{domain}/api/"
+
+
+def get_dashboard_json_from_terraform():
+    logging.info('Obtain Grafana dashboard definition from terraform')
+    cmd = f"cd {config.project_root}/terraform && terraform output grafana_dashboard"
+    p = subprocess.Popen(cmd, stdout=subprocess.PIPE, shell=True)
+    out, err = p.communicate()
+    return json.loads(out)
+
+
+def update_dashboard(base_url, dashboard):
+    logging.info('Update Grafana dashboard')
+    url = base_url + "dashboards/db"
+    body = '{"dashboard":' + json.dumps(dashboard) + ', "overwrite": true}'
+    response = requests.post(
+                url,
+                data=body,
+                headers={
+                    'Content-Type': 'application/json',
+                    'Accept': 'application/json'
+                }
+            )
+    return response
+
+
+if __name__ == '__main__':
+    main() if config.enable_cloudwatch_alarms else None
```

Comment:
This is an abuse of ternary expressions. Please don't.

Reply:
This means that in order to enable CloudWatch alarms, I have to provide Grafana credentials and endpoint. 

1) I am not sure it is correct to couple them but I want to hear your reasoning on this.

2) If it were correct, that dependency should be documented somewhere.